operator/identity/cell.go

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package identity
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/hive/cell"
+)
+
+const (
+	// GCInterval is the interval in which allocator identities are
+	// attempted to be collected
+	GCInterval = "identity-gc-interval"
+
+	// GCRateInterval is the interval used for rate limiting the GC of
+	// identities.
+	GCRateInterval = "identity-gc-rate-interval"
+
+	// GCRateLimit is the maximum identities used for rate limiting the
+	// GC of identities.
+	GCRateLimit = "identity-gc-rate-limit"
+
+	// HeartbeatTimeout is the timeout used to GC identities from k8s
+	HeartbeatTimeout = "identity-heartbeat-timeout"
+)
+
+// GCCell is a cell that implements a periodic Cilium identities
+// garbage collector.
+// The GC subscribes to identities events to mark all the related identities
+// as alive. If an identity has no activity for a prolonged interval,
+// it is first marked for deletion and eventually deleted.
+var GCCell = cell.Module(
```

Comment:
Are there other cells? Can this be just `Cell`? Same for everything else with `GC` prefix, though maybe the package should be renamed to identitygc.